
<launch>
    <node pkg="villa_octomap_server" type="octomap_object_server_node" name="octomap_server">
        <param name="resolution" value="0.02" />
        <param name="latch" value="false" />

        <!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
        <param name="frame_id" type="string" value="map" />

	    <param name="height_map" value="false" />
        
        <!-- maximum range to integrate (speedup!) -->
        <param name="sensor_model/max_range" value="2.0" />
        
        <!-- data source to integrate (PointCloud2) -->
        <remap from="cloud_in" to="/hsrb/head_rgbd_sensor/depth_registered/rectified_points" />
    
    </node>
</launch>
